src/lib/components/ui/input/input.svelte

```
@@ -20,7 +20,8 @@
 		disabled = false,
 		variant = 'regular',
 		icon,
-		button
+		button,
+		onchange = (value: string) => {}
```

ESLint

[error] 'value' is defined but never used.

(@typescript-eslint/no-unused-vars)